src/site/_includes/components/Banner.js

```diff
@@ -33,6 +33,7 @@ function Banner(content, type = 'info') {
   return `<div role="banner" class="banner ${utilityClasses}">
   <div class="banner__content flow">
     ${md.renderInline(content)}
+    <span data-banner-close-btn>Dismiss</span>
```

Comment:
While we are at it, we should inline this into the njk template and remove the `{% Banner %}` shortcode as it's none, following the pattern that shortcodes should be allowed to used by authors in their Markdown files.

`Dismiss` needs to go through the `i18n` filter.